### PR TITLE
Improve quiz styling

### PR DESCRIPTION
**Pull request by bumbu**

### CODE REVIEW

File: app/assets/stylesheets/application.css.sass [hunk 1]
```diff
@@ -9,8 +9,17 @@
...
   position: fixed
-  width: 100%
\ No newline at end of file
+  width: 100%
+
+// Add spacing around nav tabs in panel
+.panel
+  >.nav.nav-tabs
+    padding-top: 20px
+    margin-bottom: 10px
```
Comment (eugeniu-u) [inline app/assets/stylesheets/application.css.sass:22]: Line 13 makes the breadcrumbs to be separated from the tabs.
It's not an issue, but i'm not sure if they look better being separated.

File: app/assets/stylesheets/application.css.sass [hunk 2]
```diff
@@ -9,8 +9,17 @@
...
+
+  >.panel-body
+    padding: 15px 0
```
Comment (eugeniu-u) [inline app/assets/stylesheets/application.css.sass:25]: ~~On the student list page, the form for adding students is placed to the extreme left of the panel.~~
(Already fixed by pulling the form out from the panel-body scope)

File: app/assets/stylesheets/application.css.sass (hunk 1 above)
Comment (bumbu) [inline app/assets/stylesheets/application.css.sass:22]: It seemed to look better when separated. But you can revert this change if you want.

File: app/assets/stylesheets/application.css.sass (hunk 2 above)
Comment (bumbu) [inline app/assets/stylesheets/application.css.sass:25]: From what I saw in many places there are cols (ex. `.col-sm-6`) that are not inside of a `.row`. 
Bootstrap relies very much on cols being inside rows. Maybe that was the issue.

**Comment by bumbu** [comment]
You can check how changes look before merging a pull request. 
Simply pull git changes (`git pull` should be enough) and switch to the branch that corresponds to pull request (in this case it was `quiz-style-fixes`). 

This way you can check changes locally and decide if it is good to go or not. 
Also you can add changes to that branch and push them to github. This way you'll update the pull request.

File: app/assets/stylesheets/application.css.sass (hunk 2 above)
Comment (eugeniu-u) [inline app/assets/stylesheets/application.css.sass:25]: I've just looked into it again, and the problem was that i didn't put any `col-sm-...` inside `row`
